symengine/lib/symengine_wrapper.pxd

```diff
@@ -1,5 +1,10 @@
 cimport symengine
 from symengine cimport RCP, map_basic_basic, rcp_const_basic
+from libcpp.vector cimport vector
+from libcpp.string cimport string
+from libcpp cimport bool as cppbool
+
+include "config.pxi"
```

Comment:
This file needs to be installed right? If so, I would change the name of this file to symengine_config.pxi

Reply:
Nvm. Its already used in symengine.pxd